projects/batfish-client/src/test/java/org/batfish/client/ClientTest.java

```diff
@@ -251,26 +251,10 @@ public void testValidJsonPath() throws IOException {
       Client.validateJsonPath(mapper.readTree(validJsonPath));
    }
 
-// ratul commented out this test
-//     parseParaValue should allow strings like "I am not valid" to parse.
-//     i can't think of what that new function will not parse
-//       TODO: remove test or give it some teeth
-//   // Tests for parseParaValue method
-//   @Test
-//   public void testParseInvalidJsonContent() {
-//      String invalidJsonContent = "I am not valid";
```

Comment:
@ratulm I think this function does the different thing. After we read the question json file, and stored variable name and its value into the map, this function just check the value of each variable is valid json value. example: 
{
    "variable" : invalid value
}
this should be invalid, since a string in json must inside "".
Sorry for the unclear function name, I named it based on the parameter name in origin code, it actually should be something like validateVariableValue(value).